internal/cmd/grants.go

```diff
@@ -90,7 +91,7 @@ func newGrantsListCmd(cli *CLI) *cobra.Command {
 				listReq.Group = group.ID
 			}
 
-			grants, err := listAll(client.ListGrants, listReq)
+			grants, err := listAll(client.ListGrantsOld, listReq)
```

Comment:
Why not use the new ListGrants here?

Reply:
`listAll` requires a function that doesn't accept context.Context. I fixed that in e44127f138b450d1322479d29cfc8a9eb24ee69c so that we do now use `ListGrants` here.